### ISSUE TITLE
something about videopose3D

### ISSUE THREAD

**Issue by Ared521** [open-mmlab/mmpose#1801]
Is the data used in the training of videopose3d pre-trained by preprocessing_h36m.py?

**Comment by ly015** [comment]
Yes, the script `preprocessing_h36m.py` is used for processing Human3.6M data for VideoPose3D training. The raw data need to be downloaded from the official site.

**Comment by Ared521** [comment]
> Yes, the script `preprocessing_h36m.py` is used for processing Human3.6M data for VideoPose3D training. The raw data need to be downloaded from the official site.

Thank you for your reply。

**Comment by Ared521** [comment]
> Yes, the script `preprocessing_h36m.py` is used for processing Human3.6M data for VideoPose3D training. The raw data need to be downloaded from the official site.

Hi, thank you for your great work, now, I have a question:
I want to predict 3d pose in my own video, and I use body3d_two_stage_video_demo.py. However, I ran into a very tricky problem. The posetrack18 dataset website was down and I couldn't find the author's email address. I want to get to the final pose by multi frames, which is --use_multi_frames. What should I do now, please? Can JHMDB dataset be used instead? Is there any other way?
I'm looking forward to your early reply.Thank you.
